packages/reactotron-core-client/src/reactotron-core-client.ts

```diff
@@ -291,6 +313,30 @@ export class Client {
     // chain-friendly
     return this
   }
+
+  onCustomCommand(command: string, handler: () => void): () => void {
+    // Create this command handlers object
+    const customHandler: CustomCommand = {
+      id: this.customCommandLatestId,
+      command,
+      handler,
+    }
+
+    // Increment our id counter
+    this.customCommandLatestId += 1
+
+    // Add it to our array
+    this.customCommands.push(customHandler)
```

Comment:
We track custom commands on client AND server. On client we have all the metadata + the handler but on server we only have metadata. The way I just shovel over each registration/unregister over to the server individually could lead to a desync between the client and server. Might want to consider just shipping all metadata of all registered commands every single time.... but then that bloats the payload. I don't know what way is the lesser of two evils. Why must keeping things in sync be so hard?